pkg/util/constant.go

```diff
@@ -148,8 +148,8 @@ var (
 	// CstorReplicaColumnDefinations stores the Table headers for CVR Details
 	CstorReplicaColumnDefinations = []metav1.TableColumnDefinition{
 		{Name: "Name", Type: "string"},
-		{Name: "Total", Type: "string"},
-		{Name: "Used", Type: "string"},
+		{Name: "ZFS Used(compressed)", Type: "string"},
+		{Name: "LogicalReferenced", Type: "string"},
```

Comment:
These two headings are suggestions for [this output](https://github.com/openebs/openebsctl/issues/135) please suggest better words which are concise but self-explanatory.